CRM/Civicase/Hook/PreProcess/CaseCategoryWordReplacementsForChangeCase.php

```diff
@@ -50,7 +50,7 @@ private function addWordReplacements(CRM_Core_Form $form) {
    *   Form Object.
    */
   private function setPageTitle(CRM_Core_Form $form) {
-    $pageTitle = ts($form->get_template_vars('activityTypeName'));
+    $pageTitle = $form->get_template_vars('activityTypeName');
```

Comment:
Please note that this also serves a purpose. Without the translation here. Existing functionality will be broken and page titles will not be translated for the respective case category.

Reply:
Fixed, created a new function as discussed.